pkgs/tools/virtualization/multipass/default.nix

```diff
@@ -70,8 +70,8 @@ stdenv.mkDerivation
 
     # Patch the patch of the OVMF binaries to use paths from the nix store.
     substituteInPlace ./src/platform/backends/qemu/linux/qemu_platform_detail_linux.cpp \
-      --replace "OVMF.fd" "${OVMF.fd}/FV/OVMF.fd" \
-      --replace "QEMU_EFI.fd" "${OVMF.fd}/FV/QEMU_EFI.fd"
+      --replace "OVMF.fd" "${OVMF.firmware}" \
```

Comment:
Have you checked how `multipass` is using these variables?

Reply:
https://github.com/canonical/multipass/blob/f51d0ccf83b42e1e6a64fbfa8de9b7602d45192e/src/platform/backends/qemu/linux/qemu_platform_detail_linux.cpp#L175-L184

_oh no_

Reply:
OK I guess we can drop that change.